webapi/tct-colors-css3-tests/tests.full.xml

```diff
@@ -1295,6 +1103,214 @@
+        </description>
+        <specs>
+          <spec>
+            <spec_assertion element_type="property" element_name="opacity" interface="CSS" specification="CSS Color Module Level 3" section="DOM, Forms and Styles" category="Tizen W3C API Specifications"/>
+            <spec_url>http://www.w3.org/TR/2011/REC-css3-color-20110607/#transparency</spec_url>
+            <spec_statement/>
+          </spec>
+        </specs>
+      </testcase>
+      <testcase component="W3C_HTML5 APIs/DOM, Forms and Styles/CSS Color Module Level 3" execution_type="manual" id="css3_colors_tests_entry7" priority="P2" purpose="Check that opacity: (greater than 1.0) is treated to 1.0" status="approved" type="compliance">
+        <description>
+          <test_script_entry>/opt/tct-colors-css3-tests/colors/csswg/t32-opacity-clamping-1.0-b.htm</test_script_entry>
+          <bdd_test_script_entry test_script_expected_result="0">/opt/tct-colors-css3-tests/css3-color-app/testscripts/t32-opacity-clamping-1.0-b.feature</bdd_test_script_entry>
+        </description>
+        <specs>
+          <spec>
+            <spec_assertion element_type="property" element_name="opacity" interface="CSS" specification="CSS Color Module Level 3" section="DOM, Forms and Styles" category="Tizen W3C API Specifications"/>
+            <spec_url>http://www.w3.org/TR/2011/REC-css3-color-20110607/#transparency</spec_url>
+            <spec_statement/>
+          </spec>
+        </specs>
+      </testcase>
+      <testcase component="W3C_HTML5 APIs/DOM, Forms and Styles/CSS Color Module Level 3" execution_type="manual" id="css3_colors_tests_entry8" priority="P2" purpose="Check that opacity specifies compositing the contents as a single offscreen buffer." status="approved" type="compliance">
+        <description>
+          <test_script_entry>/opt/tct-colors-css3-tests/colors/csswg/t32-opacity-offscreen-b.htm</test_script_entry>
+          <bdd_test_script_entry test_script_expected_result="0">/opt/tct-colors-css3-tests/css3-color-app/testscripts/t32-opacity-offscreen-b.feature</bdd_test_script_entry>
+        </description>
+        <specs>
+          <spec>
+            <spec_assertion element_type="property" element_name="opacity" interface="CSS" specification="CSS Color Module Level 3" section="DOM, Forms and Styles" category="Tizen W3C API Specifications"/>
+            <spec_url>http://www.w3.org/TR/2011/REC-css3-color-20110607/#transparency</spec_url>
+            <spec_statement/>
+          </spec>
+        </specs>
+      </testcase>
+      <testcase component="W3C_HTML5 APIs/DOM, Forms and Styles/CSS Color Module Level 3" execution_type="manual" id="css3_colors_tests_entry11" priority="P2" purpose="Check that alpha within the offscreen buffer is composited correctly" status="approved" type="compliance">
+        <description>
+          <test_script_entry>/opt/tct-colors-css3-tests/colors/csswg/t32-opacity-offscreen-with-alpha-c.htm</test_script_entry>
```

Comment:
the html path is wrong for these cases automated by bdd, please update all of them